src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -247,6 +248,13 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       System.err.println("Compiler runtime: " + (stop - start) + "ms.");
     }
 
+    try {
+      PrintWriter writer = new PrintWriter(ops.statsfile, "UTF-8");
+      writer.println(Long.toString(stop - start));
+      writer.close();
+    } catch (Throwable ex) {
+      throw new RuntimeException("YOLO", ex);
```

Comment:
At the time of writing this I figured that if the file fails to write then there must be a show-stopping issue with the build.
- Should the build continue if we can't write the file?
- YOLO might not cut is as the error message :)

Reply:
could we catch IOException instead?

Reply:
I think so. Do we want to catch and continue the build or do we want to rethrow it with a better message?